tests/pipette/test_orchestrator_dispatch.py

```
@@ -0,0 +1,118 @@
+"""Tests for orchestrator dispatch decisions — Chunk F (F11–F13)."""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)